diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/kdoc/KdocMethods.kt

```diff
@@ -101,6 +102,8 @@ class KdocMethods : Rule("kdoc-methods") {
         if (returnCheckFailed) handleReturnCheck(node, name, kDoc, kDocTags)
         if (throwsCheckFailed) handleThrowsCheck(node, name, kDoc, missingExceptions)
 
+        if (kDocTags != null && kDocTags.isNotEmpty()) checkKDocParam(node, kDocTags)
```

Comment:
I'd rather not create a new function for this check, but merge it with `getMissingParameters` call and `handleParamCheck`.